app/views/pages/home.html.erb

```diff
@@ -18,30 +18,30 @@
 
 
     <!-- BEGIN SEARCH/ADD MODAL HIDDEN-->
-    <div id="searchAddModal" class="" style="display: none">
+    <div id="searchAddModal" aria-label="search or add" class="" style="display: none">
       <div style="visibility:visible;z-index:900;position: fixed; display: inline-block;top: 25%;left: 30%;height:auto;width:500px" class="table-wrapper border-radius-small">
         <div class="bgblur">
           <div class="table-cell text-center">
             <div class="landingfront pagenotfound">
               <div>
                 <div class="top">
                   <h1 class="" style="margin:0.67em 0 -0.3em;">Search/Add</h1>
-                  <a class="fa-stack fa-2x icon-back">
+                  <div class="fa-stack fa-2x icon-back" role="button" onClick="onArrowLeftClick()">
                     <i class="fa fa-circle fa-stack-2x blue"></i>
                     <i class="fa fa-long-arrow-left fa-stack-1x fa-inverse"></i>
-                  </a>
+                  </div>
                 </div>
                 <form class="big-box">
-                  <a class="fa-stack fa-2x icon-home" style="top:52%;z-index:1500" type="submit" name="commit" value="Create Client" data-disable-with="Create Client">
-                    <i class="fa fa-circle fa-stack-2x yellow"></i>
-                    <i id="landsearchicon" class="fa fa-search fa-stack-1x fa-inverse"></i>
-                  </a>
+                  <div class="fa-stack fa-2x icon-search" style="top:52%;z-index:1500" aria-label="search or add name" role="button" onClick="onSearchClick()">
+                    <span class="fa fa-circle fa-stack-2x yellow"></span>
+                    <span id="landsearchicon" class="fa fa-search fa-stack-1x fa-inverse"></span>
+                  </div>
```

Comment:
Using presentational components as children of <a>. Opting out for a `div` with `role="button"` as recommended below.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#all_descendants_are_presentational.